x-pack/metricbeat/module/istio/proxy/_meta/docs.asciidoc

```diff
@@ -0,0 +1,33 @@
+This is the proxy metricset of the module istio.
+This metricset collects metrics from the Envoy proxy's Prometheus exporter for Istio versions higher than 1.5
+
+Tested with Istio 1.7
+
+[float]
+=== Deployment
+
+Istio-proxy is a sidecar container that is being injected into every Pod that is
+being deployed on a Kubernetes cluster which's traffic is managed by Istio.
+Because of this reason, in oder to collect metrics from this sidecars we need
+to automatically identify these sidecar containers and start monitoring them
+using their IP and the predifined port (15090). This can be achieved easily by
+defining the proper autodiscover provider that will automatically identifies
+all these sidecar containers and will start `proxy` metricset for each one of
+them. Here is an example configuration that can be used for that purpose:
+
+
+["source", "yaml"]
+--------------------------------------------
+metricbeat.autodiscover:
+  providers:
+    - type: kubernetes
+      include_annotations: ["prometheus.io.scrape"]
```

Comment:
I see you are not using this at all, is this line needed?

Reply:
nope it's not! I will remove it to avoid for confusion 👍

Reply:
sorry I saw now this is a backport, we will need to remove it from master too

Reply:
sure thing!